### ISSUE TITLE
[Bug] error TS7016: Could not find a declaration file for module 'graphql-type-json'

### ISSUE THREAD

**Issue by galkin** [Urigo/graphql-scalars#112]
```
> tsc --project tsconfig.json
node_modules/graphql-scalars/dist/esnext/resolvers/index.d.ts:27:70 - error TS7016: Could not find a declaration file for module 'graphql-type-json'. '/Users/galkin/Projects/gamaya/web-server/node_modules/graphql-type-json/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/graphql-type-json` if it exists or add a new declaration (.d.ts) file containing `declare module 'graphql-type-json';`
27 import { GraphQLJSON as JSON, GraphQLJSONObject as JSONObject } from 'graphql-type-json';
```

At application level, it can be fixed by `npm install -D @types/graphql-type-json`
This package has defenition for that https://github.com/Urigo/graphql-scalars/blob/master/types/graphql-type-json.d.ts

Could you fix?

**Comment by ardatan** [comment]
@galkin You can try with canary version `1.0.3-ed42054.0 `.

**Comment by galkin** [comment]
@ardatan, i tested 1.0.3-ed42054.0 It works as expected.
Maybe, it's good idea mark 1.0.2 as [deprecated](https://docs.npmjs.com/cli/deprecate) because of this bug.

Anyway could you publish new version?

**Comment by ardatan** [comment]
Available in v1.0.3!
